client/me/help/help-flow/index.jsx

```diff
@@ -0,0 +1,226 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+import classnames from 'classnames';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Dialog from 'components/dialog';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import Main from 'components/main';
+import HeaderCake from 'components/header-cake';
+import SiteSelector from 'components/site-selector';
+import SegmentedControl from 'components/segmented-control';
+import DocumentHead from 'components/data/document-head';
+import MeSidebarNavigation from 'me/sidebar-navigation';
+import hasJetpackSites from 'state/selectors/has-jetpack-sites';
+import userHasAnyAtomicSites from 'state/selectors/user-has-any-atomic-sites';
+import { getCurrentUserSiteCount } from 'state/current-user/selectors';
+import userHasAnyPaidPlans from 'state/selectors/user-has-any-paid-plans';
+import getJetpackSites from 'state/selectors/get-jetpack-sites';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class HelpFlow extends Component {
+	state = {
+		displayDialog: false,
+	};
+
+	handleDialogClosure = event => {
+		this.setState( {
+			displayDialog: false,
+			blogIssueDialog: false,
+			jetpackDialog: false,
+			wooDialog: false,
+		} );
+	};
+
+	handleJetpackButtonClick = event => {
+		this.setState( { displayDialog: true, jetpackDialog: true } );
+	};
+
+	handleWooCommerceButtonClick = event => {
+		this.setState( { displayDialog: true, wooDialog: true } );
+	};
+
+	handleBlogIssueButtonClick = event => {
+		this.setState( { displayDialog: true, blogIssueDialog: true } );
+	};
+
+	siteFilter = site => {
+		const isAtomic = get( site, 'options.is_automated_transfer', false );
+
+		return site.jetpack && ! isAtomic;
+	};
+
+	backToHelp = () => {
+		page( '/help' );
+	};
+
+	render() {
+		const {
+			siteCount,
+			jetpackSiteCount,
+			hasAtomicSites,
+			hasJetpackSites,
+			hasPaidPlans,
+			translate,
+		} = this.props;
+
+		const { displayDialog, jetpackDialog, wooDialog, blogIssueDialog } = this.state;
+
+		const hasWpComSites = siteCount - jetpackSiteCount > 0;
+
+		if ( ! hasJetpackSites && ! hasAtomicSites ) {
+			page( `/help/contact/form` );
+		}
+
+		const dialogClassNames = classnames( 'help-flow__dialog', {
+			'has-site-selector': blogIssueDialog && hasJetpackSites && hasWpComSites,
+		} );
+
+		let dialogText;
+		let buttonLink;
+		let buttonText;
+
+		if ( jetpackDialog ) {
+			buttonLink = 'https://jetpack.com/contact-support/';
+			buttonText = translate( 'Contact Jetpack support' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **36** times:
`translate( 'Contact Jetpack Support.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).